infra/base-images/base-runner/run_fuzzer

```diff
@@ -67,6 +67,11 @@ rm -rf $CORPUS_DIR && mkdir $CORPUS_DIR
 rm -rf $FUZZER_OUT && mkdir $FUZZER_OUT
 
 SEED_CORPUS="${FUZZER}_seed_corpus.zip"
+
+if [ ! -z ${DO_NOT_USE_CORPUS:-} ]; then
```

Comment:
This can be removed if we use SKIP_SEED_CORPUS above in reproduce file.

Reply:
done